challenge_3/python/lex32pem/challenge_3.py

```diff
@@ -0,0 +1,58 @@
+# Challenge # 3
+# 2017 Programming Challenge @ Slack
+# 
+# Majority Element
+# 
+# Premise
+# 
+# Given an array of size n, find the majority element. The majority element is the element
+# that appears more than ⌊ n/2 ⌋ times.
+# 
+# You may assume that the array is non-empty and the majority element always exist
+# in the array.
+# 
+# for example, given:
+# 
+# array = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7]
+# 
+# your program should return 7
+# 
+# Language: Python 3.6
+
+def find_Max(u):
+    dictMax = {}                            # empty dictionary
+    max = ('', 0)                           # (max element, max occurrences)
+    for n in u:                             # to apply condition on each item of array in the defined function
+        if n in dictMax:                    # if item already exist in dictionary
+            dictMax[n] += 1                 # add 1 to the value of item [n] of array
```

Comment:
Fantastic Commenting! The idea with python is that the code can be self-documenting so using variable names that clearly describe their usage helps out a lot and makes it less code you have to write.